lingua_franca/__init__.py

```diff
@@ -1,6 +1,15 @@
-from .internal import get_default_lang, set_default_lang, get_default_loc, \
-    get_active_langs, _set_active_langs, get_primary_lang_code, \
-    get_full_lang_code, resolve_resource_file, load_language, \
-    load_languages, unload_language, unload_languages, get_supported_langs
+### DO NOT CHANGE THIS IMPORT ORDER ###
+from .internal import get_active_langs, get_supported_locs, \
+    get_full_lang_code, get_supported_langs, get_default_loc, \
+    get_primary_lang_code
```

Comment:
I'm all for standardizing. `loc` is the only one I didn't inherit.

`get_full_lang_code()` is an existing function which returns "the" full code corresponding to a primary lang code. This can't be removed wholesale, but it doesn't serve the whole purpose, and it doesn't serve "nondefault" locales in any way.

`get_default_loc()` returns the *current* "default" full code for a given primary code, which is configurable. On your rig, for instance, `get_full_lang_code('en')` will always return `en-us`, but I imagine `get_default_loc('en')` will return `en-au`.

I was thinking 'locale', but I didn't give it too much thought. I am by no means married to the names.